spec/controllers/admin/searchgov_domains_controller_spec.rb

```diff
@@ -1,21 +1,57 @@
 # frozen_string_literal: true
 
 describe Admin::SearchgovDomainsController do
-  fixtures :users, :searchgov_domains
-  let(:config) { described_class.active_scaffold_config }
+  fixtures :users, :searchgov_domains, :searchgov_urls
+
+  let(:basic_domain) { searchgov_domains(:basic_domain) }
+
+  before do
+    activate_authlogic
+    UserSession.create(users(:affiliate_admin))
+  end
+
+  describe 'after_create_save' do
+    it 'sets flash[:info] message correctly' do
+      expect { post :create, params: { record: { domain: 'www.usa.gov' } } }.
+        to change { SearchgovDomain.count }.by(1)
+
+      # Validate flash message
+      expect(flash[:info]).to include(SearchgovDomain.last.domain.to_s)
+    end
+  end
 
   describe '#update' do
-    before do
-      activate_authlogic
-      UserSession.create(users(:affiliate_admin))
+    it 'contains the specified columns' do
+      expect(described_class.active_scaffold_config.update.columns.to_a).to include(:js_renderer)
+    end
+  end
+
+  describe '#confirm_delete' do
+    it 'renders the delete_domain template' do
+      get :confirm_delete, params: { id: basic_domain.id }
+
+      expect(response).to render_template(:delete_domain)
+    end
+  end
+
+  describe '#delete_domain' do
+    context 'with correct confirmation text' do
+      it 'enqueues the deletion job and redirects to index' do
+        delete :delete_domain, params: { id: basic_domain.id, confirmation: 'DESTROY DOMAIN' }
+
+        expect(response).to redirect_to(action: :index)
+        expect(flash[:success]).to eq("Deletion has been enqueued for #{basic_domain.domain}")
```

Comment:
In general is better to validate against the `i18n` translation just because if the text in the translation changes we won't need to update the tests as well. e.g.

```ruby
 expect(flash[:success]).to eq(i18n.t('key.', domain: domain))
```